exporter/stats/stackdriver/stackdriver.go

```diff
@@ -91,9 +91,9 @@ func NewExporter(o Options) (*Exporter, error) {
 		return nil, err
 	}
 	e := &Exporter{
-		c:        client,
-		o:        o,
-		measures: make(map[stats.Measure]struct{}),
+		c:               client,
+		o:               o,
+		registeredViews: make(map[string]stats.View),
```

Comment:
map[string]struct{} is enough, we don't need to access the view.

Reply:
Done.